ProcessMaker/processmaker
# feature/FOUR-15685

## Issue & Reproduction Steps

Replace the Request for Cases

## Solution
- List the changes you've introduced to solve the issue.

## How to Test
- Login 
- Review the Top Menu -> Cases
- Click on Cases
- Review the counters show: My Cases - In Progress - Completed -  All Cases
- Review the menu left: My Cases - In Progress - Completed -  All Cases
- Update My Request and All Request
- Task list -> Update the option “Open Request” to “Open Case”
- Launchpad -> List -> Update "My Request" should change to "My Cases"
- Launchpad -> List -> Update the message for search cases

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-15685
- https://processmaker.atlassian.net/browse/FOUR-15700
- https://processmaker.atlassian.net/browse/FOUR-15701
- https://processmaker.atlassian.net/browse/FOUR-15703
- https://processmaker.atlassian.net/browse/FOUR-15704
- https://processmaker.atlassian.net/browse/FOUR-15749
- 

## Code Review Checklist

ci:deploy
ci:next
ci:screen-builder:feature/FOUR-15760
